File: python/exercício_69.py
vetor = []
vezes = []
c = 0
total =0
while c in range(15):
    vetor.append(int(input("Digite um valor para a  pocisao {}: ".format(c))))
    c +=1
vezes.append(int(input("Digite o numero que deseja verificar: ")))

c = 0
print("=-"*60)
print("Os numers digitados foram: ")
for c in range(15):
    print(vetor[c], end='')
    c = c + 1
    print()
c =0 

print(set(vetor) & set(vezes))
      
   

File: java/exercício_69.java
import java.util.Scanner;

public class exercício_69 {
    public static void main(String[] args) {
        try (Scanner numero = new Scanner(System.in)) {
            int n = 15;
            int a = 1;
            int vetor[] = new int[n];
            int vezes[] = new int[a];

            for (int c = 0; c < n; c++) {
                System.out.printf("Digite um valor para a posição %d: ", (c + 1), n);
                vetor[c] = numero.nextInt();
            }
            System.out.printf("Os numeros digitados foram:");
            for (int mostra = 0; mostra < n; mostra++) {
                System.out.println(vetor[mostra]);
            }

            System.out.printf("Digite o numero que deseja verificar: ");
            vezes[a] = numero.nextInt();

            if (vezes.equals(vetor)) {
                System.out.printf("o numero: " + vezes + "se encontra nos numeros digitados");
            } else {
                System.out.println("Numero não encontrado");
            }
        }
    }
}